growth-fancy/index.php

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+			<LINK href="style.css?version=9345" rel="stylesheet" type="text/css">
+			<!--<link rel="icon" href="favicon.gif"  type="image/gif"/>-->
+			<link rel="icon" href="favicon.png" />
+			<title>FAF growth</title>
+		</meta>
+
+		<script language="javascript" type="text/javascript" src="jquery.js"></script>
+		<script language="javascript" type="text/javascript" src="jquery.flot.js"></script>
+		<script language="javascript" type="text/javascript" src="jquery.flot.stack.js"></script>
+		<script language="javascript" type="text/javascript" src="jquery.flot.time.js"></script>
+		
+		<script language="javascript" type="text/javascript" src="script.js"></script> 
+	</head>
+	
+	<body>
+		<div id="description" style="text-align:center;">
+			<h2>FAF growth</h2>
+			<p>
+				Last <input type="text" placeholder="(optional)" id="records" value="7"></input> days.
+				<button id="generateButton" onclick="getStatsFromDocument()">Generate Chart</button>
```

Comment:
Given the general tone of those tools, you might prefer put friendly notices instead of cold factual informations. I've used wordings like "Show me!" or "Let's see" on such buttons, for example.

Reply:
I’m German, so what do you expect? 😜